nimbus/src/client/http_client.rs

```diff
@@ -91,42 +113,75 @@ mod tests {
     #[test]
     fn test_get_experiments_from_schema() {
         viaduct_reqwest::use_reqwest_backend();
-        let body = r#"
-        { "data": [
-            {
+        // There are two experiments defined here, one has a "newer" schema version
+        // in order to test filtering of unsupported schema versions.
+        let body = format!(
+            r#"
+        {{ "data": [
+            {{
+                "schemaVersion": "{}.0.0",
```

Comment:
Since the thing being formatted is such a long string, you might find it useful to use [named format arguments](https://doc.rust-lang.org/std/fmt/#named-parameters) here. Having this one as `"schemaVersion": "{current_version}.0.0"` and the other one as `"schemaVersion": "{newer_version}.0.0"` might make the test a little easier to understand for future readers.